frontend/src/components/elements/Text/styled-components.ts

```diff
@@ -21,4 +21,9 @@ export const StyledText = styled.div(({ theme }) => ({
   whiteSpace: "pre",
   fontSize: theme.fontSizes.sm,
   overflowX: "auto",
+  display: "flex",
+  visibility: "visible",
+  verticalAlign: "middle",
+  flexDirection: "row",
+  alignItems: "center",
```

Comment:
Nit: these styling additions should no longer be necessary after Text.tsx refactor

Reply:
True, it's updated now 👍